test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareServiceOptionsTests.cs

```diff
@@ -266,7 +266,7 @@ private async Task<Mock<RequestContext<SchemaCompareResult>>> SendAndValidateSch
                 };
 
                 SchemaCompareGenerateScriptOperation generateScriptOperation1 = new SchemaCompareGenerateScriptOperation(generateScriptParams1, schemaCompareOperation1.ComparisonResult);
-                generateScriptOperation1.Execute();
+                generateScriptOperation1.Execute(TaskExecutionMode.Script);
```

Comment:
for my understanding - Does the TaskExecutionMode.Script do anything different (I see it setting the task name - but anything else?)

Reply:
This is how ADS knows to open the script after the task is finished